mvc/src/test/java/nextstep/mvc/DispatcherServletTest.java

```diff
@@ -0,0 +1,69 @@
+package nextstep.mvc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import nextstep.mvc.controller.asis.ControllerHandlerAdapter;
+import nextstep.mvc.controller.tobe.AnnotationHandlerMapping;
+import nextstep.mvc.controller.tobe.HandlerExecutionHandlerAdapter;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import samples.TestManualHandlerMapping;
+import support.FakeRequestDispatcher;
+
+class DispatcherServletTest {
+
+    @Test
+    @DisplayName("어노테이션 기반의 컨트롤러에 대한 요청을 처리할 수 있다.")
+    void serviceAnnotationController() throws ServletException {
+        // given
+        final DispatcherServlet dispatcherServlet = new DispatcherServlet();
+        final AnnotationHandlerMapping handlerMapping = new AnnotationHandlerMapping("samples");
+        handlerMapping.initialize();
+        dispatcherServlet.addHandlerMapping(handlerMapping);
+        dispatcherServlet.addHandlerAdapter(new HandlerExecutionHandlerAdapter());
```

Comment:
```suggestion
        dispatcherServlet.addHandlerMapping(handlerMapping);
        dispatcherServlet.addHandlerAdapter(new HandlerExecutionHandlerAdapter());
        dispatcherServlet.init();
```
이게 더 테스트의 목적에 부합하지 않을까요?!

Reply:
변경했습니다 감사합니다~